src/SFA.DAS.EAS.Account.Worker/Infrastructure/ServiceLocator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using StructureMap;
+
+namespace SFA.DAS.EAS.Account.Worker.Infrastructure
+{
+	public static class ServiceLocator
+	{
+		private static IContainer _container;
+
+		public static void Initialise(IContainer container)
+		{
+			_container = container;
+		}
+
+		public static T Get<T>()
+		{
+			return _container.GetInstance<T>();
+		}
+
+		public static T Get<T>(Type type) where T : class
+		{
+			return _container.GetInstance(type) as T;
+		}
+
+		public static IContainer CreateNestedContainer()
+		{
+			return _container.GetNestedContainer();
+		}
+
+		public static void Register<T>(IContainer container, object instance) where T : class
```

Comment:
Registering components after the container is initialised will cause the container to lock while it re-configures itself and isn't generally advised. Are we ok to remove this method?

Reply:
Done